src/components/SelectionToolbar.tsx

```diff
@@ -34,7 +35,15 @@ function isActive(props) {
   const { view } = props;
   const { selection } = view.state;
 
-  return selection && !selection.empty && !selection.node;
+  if (!selection) return false;
+  if (selection.emtpy) return false;
+  if (selection.node) return false;
+
+  const slice = selection.content();
+  const fragment = slice.content;
+  const nodes = fragment.content;
+
+  return some(nodes, n => n.content.size);
```

Comment:
this is stupid looking but it seems ok, we only want to return the floating toolbar if there is a visible selection, meaning we're selecting things beyond empty paragraph nodes

Reply:
Without running the code, I believe there should be a proxy for this in that either slice or fragment has a size?

Reply:
I couldn't find any; all the sizes come back > 0 on every level above the nodes. The thing that make me sorta nervous is the fixed hierarchy depth. I don't know if that's a stable aspect of prosemirror or if it's actually a special case that I'm hardcoding here